packages/designer-tools/src/components/InitialForm.js

```diff
@@ -0,0 +1,91 @@
+import React, { Fragment, useState } from "react";
+import { brands } from "../data";
+import { NewToken } from "./NewToken";
+import { UpdateToken } from "./UpdateToken";
+import { JsonOutput } from "./JsonOutput";
+
+export const InitialForm = () => {
+	const [selectedToken, setSelectedToken] = useState({
+		action: "update",
+		type: "Brand",
+		brand: "",
+	});
+	const [submitted, setSubmitted] = useState(false);
+	const [tokens, setTokens] = useState();
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+		setSubmitted(true);
+
+		let url =
+			'https://raw.githubusercontent.com/penske-media-corp/pmc-larva/503492cc17e1f4b5362f09ccae28209da9ee415c/packages/larva-tokens/build/' + selectedToken.brand + '.raw.json';
```

Comment:
Will this be switched to reference `master` after the PR is merged, or should this need to reference the latest release? If the latter, we can use the GH API to get the hash for the latest release.

Reply:
Yep, it will be in master! I'm opening a separate PR for that shortly.